thanks for catching that, I only saw references to `TAURI_SIGNING_PRIVATE_KEY` etc. fixing that now.

---

> Separately: I could not find the snapcraft failure in that run — it died at the AppImage step, so snapcraft pack never executed. If the missing rustup came from a Snap Store build log rather than Actions, that would be worth a line in the PR body for whoever reads this later.

the snapcraft error is found in the previous successful build `v2.7.1`: 

```
Run # Build snap
Launching managed ubuntu 24.04 instance...
Creating new instance from remote
Creating new base instance from remote
Creating new instance from base instance
Starting instance
Initializing lifecycle
Installing build-packages
Installing build-snaps
Environment validation failed for part 'markpad': 'rustup' not found and part 'markpad' does not depend on a part named 'rust-deps' that would satisfy the dependency.
Failed to run snapcraft in instance
Full execution log: '/home/runner/.local/state/snapcraft/log/snapcraft-20260806-043952.793260.log'
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -284,8 +284,8 @@ jobs:
       - name: Strip host-coupled libraries from AppImage
         if: matrix.platform == 'ubuntu-24.04'
         env:
-          TAURI_SIGNING_PRIVATE_KEY: ${{ secrets.TAURI_SIGNING_PRIVATE_KEY }}
-          TAURI_SIGNING_PRIVATE_KEY_PASSWORD: ${{ secrets.TAURI_SIGNING_PRIVATE_KEY_PASSWORD }}
+          TAURI_PRIVATE_KEY: ${{ secrets.TAURI_SIGNING_PRIVATE_KEY }}
+          TAURI_PRIVATE_KEY_PASSWORD: ${{ secrets.TAURI_SIGNING_PRIVATE_KEY_PASSWORD }}
         run: |
           set -euo pipefail
 
@@ -329,7 +329,7 @@ jobs:
           # which no longer matches. signer sign reads the key/password from the
           # env above and writes <APPIMAGE>.sig, which the upload and the
           # latest.json job below both consume.
-          npm run tauri signer sign -- "$APPIMAGE" -k "$TAURI_SIGNING_PRIVATE_KEY" -p "$TAURI_SIGNING_PRIVATE_KEY_PASSWORD"
+          npm run tauri signer sign -- "$APPIMAGE" -k
           if [ ! -f "$APPIMAGE.sig" ]; then
             echo "::error::signer sign did not produce $APPIMAGE.sig" >&2
             exit 1

